Gemfile

```
@@ -0,0 +1,19 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ruby '2.5.3'
+
+source 'https://rubygems.org/' do
+  gem 'kitchen-terraform', '~> 4.0.3'
```

This pinning should be loosened.

```suggestion
  gem 'kitchen-terraform', '~> 4.3'
```